services/net/filemonitor/Dockerfile

```diff
@@ -22,8 +22,17 @@ RUN dotnet build -c $ASPNETCORE_ENVIRONMENT -o /build
 
 FROM mcr.microsoft.com/dotnet/sdk:6.0 as deploy
 
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
```

Comment:
Fixed a number of issues here.  This file should have been tested before attempting to deploy to Openshift.